### PR TITLE
All in one installer script

### PR DESCRIPTION
**Pull request by arvin-a**
Created an all in one installer to make it easier to install Kubevirt CSI driver in an automated fashion

- Updating yamls with variables to allow for scripting
- All in one script to install kubevirt csi ./deploy/install-kubevirt-csi.sh
- Updated readme with new instructions
- Changed the node selector for the csi driver deployment from master to worker

```release-note
NONE
```

### CODE REVIEW

File: deploy/040-controller.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccount: kubevirt-csi-controller-sa
       priorityClassName: system-cluster-critical
       nodeSelector:
-        node-role.kubernetes.io/master: ""
+        node-role.kubernetes.io/worker: ""
```
Comment (davidvossel) [inline deploy/040-controller.yaml:19]: Actually, let's just remove this label entirely. THere's no reason why this controller needs to live on nodes with the master label now.

**Review by davidvossel — COMMENTED** [review]
this looks useful.

I don't know how much you want to get involved here, but if we want to merge something like this, we need the changes to work with our ci scripts

https://github.com/kubevirt/csi-driver/blob/main/hack/cluster-sync.sh
and
https://github.com/kubevirt/csi-driver/blob/main/hack/cluster-sync-split.sh

those scripts use the deploy/* yaml files today and would need to be altered to work with your templated yaml files. 

It's likely that we should be looking to move all of this to kustomize now. no one has been able to work on that yet though :/

File: deploy/040-controller.yaml (hunk 1 above)
Comment (arvin-a) [reply deploy/040-controller.yaml:19]: ok, will do.

**Comment by awels** [comment]
I am about to push a PR that uses Kustomize, should be pushed today, just running final testing.

**Comment by awels** [comment]
https://github.com/kubevirt/csi-driver/pull/74 is the kustomize support for deploying. I only have the development environment so I don't know how well it works with other environments.

**Comment by arvin-a** [comment]
Closing this PR due to the Kustomize PR already being evaluated.

**Pull request closed without merging**